docs/variable-tests.md

```diff
@@ -4,16 +4,14 @@ category: 5fdf9fc9c2a7ef443e937315
 hidden: true
 ---
 
-<Variable variable="defvar" /> and `<Variable variable="defvar" />` and:
+This is the variable `defvar`: {user.defvar}
 
-```
-<Variable variable="defvar" />
-```
+Ok, but this one is defined: {user.email}
 
-and
+It does not render in code blocks:
 
-```js
-const xyz = "<Variable variable="defvar" />";
+```
+{user.defvar}
 ```
```

Comment:
I think these vars need to be usable within code blocks; I've seen customers use this functionality to construct customized copyable snippets! (It used to be that you'd escape them– something like `\<<defvar\>>` –if you wanted to show the string literally.)